test-requirements.txt

```diff
@@ -1,2 +1,4 @@
 pytest
 coverage
+twisted
```

Comment:
Since twisted[tls] is already being installed via `install_requires` inside of `setup.py`, couldn't this be left out?